Externalize auth providers (#1055)
SAP/luigi: docs/migration-to-v1.md

<!-- meta
{
  "node": {
    "label": "Migration to 1.x",
    "category": {
      "label": "Basics"
    },
    "metaData": {
      "categoryPosition": 1,
      "position": 4
    }
  }
}
meta -->

# Migration Guides for Luigi 0.x to 1.x

<!-- accordion:start -->

### Authorization Providers

We have excluded the default authorization providers from Luigi Core and published them as separate modules.
The property `customIdpProvider` has been renamed to `idpProvider`, since you now always need to define a provider.

> Additionally you need to copy callback assets to your Core application.

Reviewer: ```suggestion
Additionally, you need to copy callback assets to your Core application.
```

> Follow the installation guide on the plugin's readme:

Reviewer: ```suggestion
To install the plugins, follow these installation guides:
```
